### ISSUE TITLE
How does it work

### ISSUE THREAD

**Issue by gesucanez** [mateuschmitz/bet365-chrome-extension#1]
Hi,

i feel stupid but...i've imported the extension but now how can i get odds ?

regards

**Comment by mateuschmitz** [comment]
Hi @gesucanez 
This a work in progress so, at this moment it's not exporting the odds yet. I expect to finish it in a few days. I'll keep this issue open and will update it when the extension is more usable.

best regards!

**Comment by gesucanez** [comment]
that's super cool ! i suggest you the possibility to export it via json and even better ( to handle with simplicity for example for me on c#) to load them always in json but locally on the browser in that way i can  get http web request from my personal page with the odds.

**Comment by mateuschmitz** [comment]
Yeah, the main purpose is to collect and send (maybe a POST) data to an endpoint specific defined by the user. How the odds change every time, this extension will exec the tasks every 5, 10 or 30 seconds and if there's something new in the odds, it sends to the endpoint.

I already did the poc and it works very well, this project will be a definitive and usable extension. There's no way to get the odds via this extension yet, sorry.

I want to work more and more quickly in this project but, currently I'm pursuing my Master's degree and my spare time is basically none :(

**Comment by gesucanez** [comment]
thank you very much for the reply buddy ! i hope you will release it soon.

**Comment by coder77** [comment]
Their odds is transmit by web socket bro.

**Comment by finanshow** [comment]
How's it going with the project?
